release/smoke-tests/run-smoke-tests.sh

```diff
@@ -35,10 +41,11 @@ function usage() {
     echo ""
     echo "Required arguments:"
     echo -e "-v TAG_NAME\tSpecify the image tag name such as '1.2'"
-    echo -e "-r REPOSITORY\tSpecify the docker repository name (ex: opensearchstaging or opensearchproject). The tag name will be pointed to '-v' value and 'latest'"
     echo ""
     echo "Optional arguments:"
     echo -e "-h\t\tPrint this message."
+    echo -e "-r REPOSITORY\tSpecify the docker repository name (ex: opensearchstaging or opensearchproject). The tag name will be pointed to '-v' value and 'latest'"
```

Comment:
Can I know why this was moved from a required to an optional argument?

Reply:
I requested this in order to be able to run smoke tests on local images. This way we can pre-test a local build before asking for a staging release.